scico/examples.py

```diff
@@ -317,3 +317,52 @@ def spnoise(img: Array, nfrac: float, nmin: float = 0.0, nmax: float = 1.0) -> A
         imgn = imgn.at[spm < nfrac - 1.0].set(nmin)
         imgn = imgn.at[spm > 1.0 - nfrac].set(nmax)
     return imgn
+
+
+def create_3D_phantom_foam(
+    im_shape: Shape,
+    N_sphere: int,
+    r_mean: float = 0.1,
+    r_std: float = 0.001,
+    pad: float = 0.01,
+    is_random: bool = False,
+):
+    """Construct a 3D phantom with random radii and centers
+
+    Args:
+        im_shape: Shape of input image.
+        N_sphere: number of spheres added
+        r_mean: Mean radius of sphere (normalized to 1 along each axis).
+                Default 0.1.
+        r_std: Standard deviation radius of sphere (normalized to 1
+                along each axis). Default 0.001.
+        pad: Padding length (normalized to 1 along each axis). Default 0.01.
+        is_random: If False, random seed is set to 1 in order to make the
```

Comment:
Some style violations.

Reply:
I fixed some style violations. Not sure if I got them all. Should 0.01 be `0.01`? I based it on the `spnoise` function.

Reply:
I don't recall if the style guide provides guidance on "0.01". If not, leave it as it is. Looks good except for missing double ticks around ``False``.

Reply:
Fixed it.